scripts/zones/The_Garden_of_RuHmet/mobs/Jailer_of_Faith.lua

```diff
@@ -7,27 +7,114 @@ mixins = { require("scripts/mixins/job_special") }
 -----------------------------------
 local entity = {}
 
+-- Jailer of Faith takes extra damage when flower is open
+local openFlower = function(mob)
+    mob:setLocalVar("PhysicalDamage", 0)
+    mob:setLocalVar("MagicalDamage", 0)
+    mob:setLocalVar("RangedDamage", 0)
+    mob:setLocalVar("BreathDamage", 0)
+    mob:delMod(xi.mod.ATTP, 10)
+    mob:setMod(xi.mod.HTH_SDT, 1250)
+    mob:setMod(xi.mod.SLASH_SDT, 1250)
+    mob:setMod(xi.mod.PIERCE_SDT, 1250)
+    mob:setMod(xi.mod.IMPACT_SDT, 1250)
+    mob:setMod(xi.mod.DMGPHYS, 112)
```

Comment:
we've got every type of physical damage -and-  the all phys dmg mod? I'm confused on the intent.

Reply:
My guess? copy paste from JoT. Probably only needs DMGPHYS